processor/internal/transformer/destination_transformer/embedded/warehouse/internal/model/model.go

```diff
@@ -0,0 +1,15 @@
+package model
+
+type (
+	SchemaType = string
+)
+
+const (
+	StringDataType   SchemaType = "string"
```

Comment:
Where are we using these new types?

Reply:
Earlier these constants were defined in [here](https://github.com/rudderlabs/rudder-server/blob/master/warehouse/internal/model/schema.go#L11-L21). Now since it is internal to the warehouse package, it can't be used. 
These are being used `datatype.go`.